ui/src/app/components/toast-banner.tsx

```diff
@@ -19,7 +19,6 @@ const styles = reactStyles({
     position: 'absolute',
     top: '0',
     right: '0',
-    zIndex: 103,
```

Comment:
We are removing this because its being passed via props?

Reply:
yes.  We need the parent components to set this value instead of toast-banner because we need these values to be different for the different toast-banners.